cw_monero/lib/monero_wallet.dart

```diff
@@ -841,10 +841,12 @@ abstract class MoneroWalletBase extends WalletBase<MoneroBalance,
   }
 
   void _askForUpdateBalance() {
-    final unlockedBalance = _getUnlockedBalance();
+    final _ub = _getUnlockedBalance();
+    final _fb =_getFrozenBalance();
+    final unlockedBalance = _ub - _fb;
     final fullBalance = monero_wallet.getFullBalance(
-      accountIndex: walletAddresses.account!.id);
-    final frozenBalance = _getFrozenBalance();
+      accountIndex: walletAddresses.account!.id) - _fb;
```

Comment:
shouldn't the fullBalance, stay as fullBalance? i.e including the frozen coins? or are we using them differently in our UI layer?